chapter2/tests/t20/terraform/base/config.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_version = ">= 1.5.0"
+
+  required_providers {
+    aws = "~> 5.1"
```

Comment:
if you use tflint on this thing it will fail with the error
for each provider you need to define the source and version.
https://github.com/terraform-linters/tflint

previously it was OK to assume the source was `hashicorp/aws` but today tflint complaints.

Reply:
> if you use tflint on this thing it will fail with the error for each provider you need to define the source and version. https://github.com/terraform-linters/tflint
> 
> previously it was OK to assume the source was `hashicorp/aws` but today tflint complaints.

@dmitry-mightydevops 
Ok, got it, done